src/tusb_option.h

```diff
@@ -171,7 +171,7 @@
 
 // Debug enable to print out error message
 #ifndef CFG_TUSB_DEBUG
-  #define CFG_TUSB_DEBUG 0
+  #define CFG_TUSB_DEBUG 2
```

Comment:
I think this update should be reverted so the debug output is off by default when not explicitly enabled.

Reply:
Well spotted!